src/plugin.c

```diff
@@ -423,28 +423,6 @@ EXPORT m64p_error CALL PluginGetVersion(m64p_plugin_type *PluginType, int *Plugi
 
 EXPORT unsigned int CALL DoRspCycles(unsigned int Cycles)
 {
-    /* Since RSP plugin API doesn't provide a "RomOpen" function
-     * we implement one with a flag inside DoRspCycle.
-     * once_per_rom is reset in RomClose. */
-    if (!g_hle.once_per_rom) {
-
```

Comment:
I guess I can just clear the list in RomClosed no?

Reply:
@bsmiles do you agree here?

Reply:
I'd prefer leaving the explicit RomOpen logic. But it would probably work with just setting count to zero in RomClose.
 Either way, it's a non blocking issue from my pov

Reply:
I guess we can always add it back if ever needed